README.md

```diff
@@ -59,12 +60,8 @@ There is also a new sub-action, `actions/upload-artifact/merge`. For more info,
 ### Breaking Changes
 
 1. On self hosted runners, additional [firewall rules](https://github.com/actions/toolkit/tree/main/packages/artifact#breaking-changes) may be required.
-2. Uploading to the same named Artifact multiple times.
```

Comment:
I would prefer to still keep (2) but perhaps change the wording a bit.

While it's not possible to upload to the same named artifact multiple times, you can overwrite it and/or merge artifacts into a single one. An example of this breaking change is outlined in [the migration guide](https://github.com/actions/upload-artifact/blob/main/docs/MIGRATION.md#multiple-uploads-to-the-same-named-artifact).

It's breaking since they still need to upload the artifacts _separately_ then merge them, or overwrite.